### ISSUE TITLE
Login - Link text should be consistent with location user is being taken

### ISSUE THREAD

**Issue by mmiron-HC** [mmiron-HC/WST-RSAMS#29]
URL: https://hcdev.openplus.ca/en/user
User Agent: Chrome/NVDA
Language: English/French

Issue Description: the link "forgot your password" should be modified to reflect the name of the page where they are being navigated to "Reset your password"

Steps to reproduce:
Enter invalid credentials and trigger "Login" 
Error appears below the input field with a "Forgot your password" link. The link navigates users to the "Reset password" page. 
![image](https://user-images.githubusercontent.com/65728700/128180193-228f97c2-7630-48a8-9193-c29c52ebdec3.png)

Expected results:
Link purpose while similar may cause confusion. It would be best to rename the link or the Reset Password tab/page heading to match.

**Comment by echouin** [comment]
@mmiron-HC from OP: "This is core functionality. It isn't an issue in WCAG, more of a cognitive issue related to UX. I'm recommending this for closure.﻿﻿"
I have marked this as resolved.

**Comment by mmiron-HC** [comment]
@echouin  this is a WCAG issue. As you pointed out, people with cognitive difficulties would be affected. This falls within WCAG.  If you choose to not address it we can set the milestone to Won't fix.

**Comment by echouin** [comment]
@mmiron-HC I am the messenger on OP's responses (when I indicate: "from OP").
I had proposed the following solution to OP back in Feb. If implemented, would this solve the issue?

EN: Reset your password?
﻿FR: Réinitialiser votre mot de passe?

﻿Change Reset password page URLs to:

﻿EN: ﻿[https://rsams.prod.cloud.openplus.ca/en/user/reset-password](https://rsams.prod.cloud.openplus.ca/fr/user/password)

FR (please let me know if this won't work, if so just use 'reset-password'): ﻿﻿[https://rsams.prod.cloud.openplus.ca/fr/user/reinitialiser-mot-de-passe](https://rsams.prod.cloud.openplus.ca/fr/user/password)﻿﻿

[Edit]()[](javascript:void(0))

**Comment by hydegeoff** [comment]
Yes, that would work, (if you replace forgot you password with your text).  Otherwise, something like this:
Unrecognized Username or Password, Forgot your password? Try [resetting it](https://rsams.dev.cloud.openplus.ca/en/user/password) (link)

**Comment by echouin** [comment]
This is now live:
![image](https://github.com/user-attachments/assets/31afc586-1a77-44c0-ac11-2ade9f3c60ce)
![image](https://github.com/user-attachments/assets/4350a4cf-916d-4adb-84fd-c9868dcb0eed)
